CHANGELOG.md

```diff
@@ -2,6 +2,144 @@
+* Remove unused class environments by @raskad in https://github.com/boa-dev/boa/pull/3332
+* Improve highlighter performance by @jedel1043 in https://github.com/boa-dev/boa/pull/3341
+* Cleanup `get_option` and calls to the function by @jedel1043 in https://github.com/boa-dev/boa/pull/3355
+* Fix new lints for Rust 1.73 by @jedel1043 in https://github.com/boa-dev/boa/pull/3361
+* Refactor compile time environment handling by @raskad in https://github.com/boa-dev/boa/pull/3365
+* Update all dependencies by @jedel1043 in https://github.com/boa-dev/boa/pull/3400
+* Optimize `shift` for dense arrays by @jedel1043 in https://github.com/boa-dev/boa/pull/3405
+* Disallow changing type of already created objects by @jedel1043 in https://github.com/boa-dev/boa/pull/3410
+* Merge `CodeBlock` constant pools by @HalidOdat in https://github.com/boa-dev/boa/pull/3413
+* Move ordinary function `[[ConstructorKind]]` to `CodeBlock` by @HalidOdat in https://github.com/boa-dev/boa/pull/3439
+* Move `FunctionKind` to `CodeBlock` by @HalidOdat in https://github.com/boa-dev/boa/pull/3440
+* Unify generator and ordinary function creation by @HalidOdat in https://github.com/boa-dev/boa/pull/3441
+* Move `arguments` object creation to bytecode by @HalidOdat in https://github.com/boa-dev/boa/pull/3432
+* Move parameter environment creation to bytecode by @HalidOdat in https://github.com/boa-dev/boa/pull/3433
+* Prevent `DefVar` opcode emit for global binding by @HalidOdat in https://github.com/boa-dev/boa/pull/3453
+* Transition `Intl` types to `NativeObject` API by @jedel1043 in https://github.com/boa-dev/boa/pull/3491
+* Migrate `Temporal` to its own crate. by @nekevss in https://github.com/boa-dev/boa/pull/3461
+* Reestructure repo and CI improvements by @jedel1043 in https://github.com/boa-dev/boa/pull/3505
+* Fix rust 1.75 lints by @raskad in https://github.com/boa-dev/boa/pull/3540
+* Remove double indirection in module types by @jedel1043 in https://github.com/boa-dev/boa/pull/3640
+* Fix clippy warnings for rustc 1.76 by @jedel1043 in https://github.com/boa-dev/boa/pull/3668
+
+## New Contributors
+* @AlvinKuruvilla made their first contribution in https://github.com/boa-dev/boa/pull/3129
+* @tunz made their first contribution in https://github.com/boa-dev/boa/pull/3109
+* @postmeback made their first contribution in https://github.com/boa-dev/boa/pull/3092
+* @kelbazz made their first contribution in https://github.com/boa-dev/boa/pull/3193
+* @frisoft made their first contribution in https://github.com/boa-dev/boa/pull/3209
+* @mattsse made their first contribution in https://github.com/boa-dev/boa/pull/3297
+* @arexon made their first contribution in https://github.com/boa-dev/boa/pull/3381
+* @johnyob made their first contribution in https://github.com/boa-dev/boa/pull/3369
+* @ahaoboy made their first contribution in https://github.com/boa-dev/boa/pull/3406
+* @tj825 made their first contribution in https://github.com/boa-dev/boa/pull/3465
```

Comment:
Looks like #3481 might be missing here. I think it was his first contribution.

Reply:
Ok I've added